src/Service/Startup.cs

```diff
@@ -798,8 +798,16 @@ private void ConfigureAuthentication(IServiceCollection services, RuntimeConfigP
                         }
```

Comment:
Given that the default authProvider is now AppService, we should always simply log a warning even if its production mode and appservice environment config is missing. The exception above may restrict usage of DAB in case of anonymous access outside of appservice environment which we should avoid. Having said that, the above exception with `message: AppServiceAuthenticationInfo.APPSERVICE_PROD_MISSING_ENV_CONFIG,` should be removed.